include/phasar/ControlFlow/CallGraphData.h

```diff
@@ -0,0 +1,33 @@
+/******************************************************************************
+ * Copyright (c) 2024 Fabian Schiebel.
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of LICENSE.txt.
+ *
+ * Contributors:
+ *     Maximilian Leo Huber and others
+ *****************************************************************************/
+
+#ifndef PHASAR_PHASARLLVM_CONTROLFLOW_CALLGRAPHDATA_H
+#define PHASAR_PHASARLLVM_CONTROLFLOW_CALLGRAPHDATA_H
+
+#include "llvm/ADT/StringRef.h"
+#include "llvm/Support/raw_ostream.h"
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace psr {
+struct CallGraphData {
+  std::unordered_map<std::string, std::vector<int>> FToFunctionVertexTy{};
```

Comment:
are there assumption with respect to the format of the key, e.g. mangled(how?)/unmangled?

also, why vector<int>? Do we really want signedness here? The name suggests the value to be a vertex id, which I would expect to be unsigned. signed/unsigned mismatch is prone to causing unexpected trouble.

Reply:
This is for unmangled names; I have clarified this as a comment.

Changed the signed int to unsigned; you are right, we don't need signedness here